controllers/storagecluster/cephtoolbox.go

```diff
@@ -81,6 +81,9 @@ func (r *StorageClusterReconciler) ensureToolsDeployment(sc *ocsv1.StorageCluste
 				toolsDeployment.Spec.Template.Spec.HostNetwork = false
 			}
 		}
+		if runOnHostNetwork(sc) {
```

Comment:
I see SCC for ocs-metrics but not for toolbox, is it misisng?

Reply:
toolbox pod uses rook-ceph-default service account and it’s able to use host network, so did not add additional scc